application/common/models/User.php

```diff
@@ -316,6 +333,7 @@ public function getAttributesForEmail()
             'isMfaEnabled' => count($this->mfas) > 0 ? true : false,
             'mfaOptions' => $this->getVerifiedMfaOptions(),
             'numRemainingCodes' => $this->countMfaBackupCodes(),
+            'inviteCode' => Invite::getInviteCode($this->id)->uuid,
```

Comment:
It looks like the call to `Invite::getInviteCode(...)` will end up creating an invite code for every user that an email is ever sent to, even when they already have a password and we have no desire to create an invite code for them. That seems worth changing.

Reply:
Oh, yikes. I'll work on that.

Reply:
Done.